### PR TITLE
SnpEffRunner Unit test

### PR DESCRIPTION
**Pull request by Mark-de-Haan**

### CODE REVIEW

**Comment by bartcharbon** [comment]
please revert changes in VCFUtils

**Comment by Mark-de-Haan** [comment]
test this please

**Comment by Mark-de-Haan** [comment]
test this please

File: molgenis-data-annotators/src/test/java/org/molgenis/data/annotation/entity/impl/SnpEffRunnerTest.java [hunk 1]
```diff
@@ -302,4 +754,29 @@ public void getInputVcfFileTest()
...
+		List<Entity> results = newArrayList(
+				snpEffRunner.getSnpEffects(entities.iterator(), new File("src/test/resources/test-snpeff.vcf")));
+
+		for (Entity resultEntity : results)
+		{
+			resultEntity.set("id", null);
+		}
+
+		assertEquals(results, expectedEffectEntities);
+	}
```
Comment (tommydeboer) [inline molgenis-data-annotators/src/test/java/org/molgenis/data/annotation/entity/impl/SnpEffRunnerTest.java]: Is it possible to split this test into multiple methods? One for single allele, one for multiple alleles, one for multiple genes and one for multiple genes and alleles? Then its a bit clearer what's being tested I think.

**Comment by Mark-de-Haan** [comment]
Merged in other branch

**Pull request closed without merging**